console/src/main/java/org/nervos/ckb/APIClient.java

```diff
@@ -32,7 +32,7 @@ public static void main(String[] args) throws IOException {
 
         System.out.println("CellOutputWithOutPoint: " + gson.toJson(RpcRequest.getLiveCell()));
 
-        System.out.println("Transaction hash: " + RpcRequest.sendTransaction(new Transaction(0, Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
+        System.out.println("Transaction hash: " + RpcRequest.sendTransaction(new Transaction(0, Collections.emptyList(), Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
```

Comment:
@duanyytop Can we extract these test print outs from `main` by moving them to separate tests or method that a test/user can call explicitly?

Reply:
Now there is a test file named APITest.java, so APIClient is unnecessary.

Please @BaojunCZ delete these print outs code.

Reply:
ok